ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -279,6 +279,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
+- as: azure-ipi-private-workers-marketimage-p2-f14
```

Comment:
I think we drop the following `azure-ipi-private-p2-f14` job now, using this one to replace that.